### ISSUE TITLE
Feature Request: Add an option in the token config for flashlights

### ISSUE THREAD

**Issue by SalieriC** [RealDeuce/torch#12]
Please add an option for flashlights. Settings as usual in the settings but a flashlight could be selected in the token options and replaces the torch if activated.

**Comment by RealDeuce** [comment]
The recent merge made the consumed item configurable... maybe that's enough for your game?

**Comment by SalieriC** [comment]
Thing is: a flashlight gives a beam not a circle of light. I don't know if and how I could configure this for your module.

**Comment by caewok** [comment]
Yes, it would be nice if the initial configuration allowed you to specify the degrees for the light, like how the default vision settings can do for a character emitting a light. So a flashlight might be 90º or 180º or something comparable, depending on user preference.

**Comment by jalensailin** [comment]
I think this feature would be awesome. In DnD5e there are torches, lanterns and hooded lanterns which all give off light differently. It would be awesome to be able to choose which one you wanted to you at any given time (maybe have multiple icons in the Token HUD?) Similar to the status effects, when you click on the torch icon it opens up another window with more options to choose from. These options could be configured in the settings to have the appropriate name for whatever system (hooded lantern vs flashlight, for example), and the appropriate emission angles/brightness.  Idk if this is way too much to implement but just a thought. Thanks for the great module so far.
